docs_user/modules/proc_verifying-the-image-service-adoption.adoc

```diff
@@ -2,19 +2,19 @@
 
 = Verifying the {image_service} adoption
 
-Verify that you successfully adopted your {image_service_first_ref} to the {rhos_long} {rhos_curr_ver} deployment.
+Verify that you adopted the {image_service_first_ref} to the {rhos_long} {rhos_curr_ver} deployment.
 
 .Procedure
 
-. Test the {image_service_first_ref} from the {rhos_prev_long} CLI. You can compare and make sure the configuration has been correctly applied to the {image_service} pods:
+. Test the {image_service} from the {rhos_prev_long} command line interface (CLI). You can compare and ensure that the configuration is applied to the {image_service} pods:
 +
 ----
-os-diff diff /etc/glance/glance.conf.d/02-config.conf glance_patch.yaml --frompod -p glance-api
+$ os-diff diff /etc/glance/glance.conf.d/02-config.conf glance_patch.yaml --frompod -p glance-api
 ----
 +
-If no line appears, then the configuration is correctly done.
+If no line appears, then the configuration is correct.
```

Comment:
What should customers do if a line appears and the configuration is incorrect?

Reply:
How can we be notified of an incorrect line? (I have no context on the `os-diff` tool and how the warnings are generated).

Reply:
@matbu Can you provide an answer? How does the os-diff tool indicate an incorrect configuration?

Reply:
My concern is that we compare a 17.x config w/ a 18 config that might be different in terms of defaults injected by the `glance-operator` (and it's totally fine). I didn't try the `os-diff` tool but I'm wondering if we're able to `ignore` part of the config that would result in false positive issues.